src/util/domain-icon.js

```diff
@@ -5,6 +5,9 @@ export default function domainIcon(domain, state) {
   case 'alarm_control_panel':
     return state && state === 'disarmed' ? 'mdi:lock-open' : 'mdi:lock';
 
+  case 'binary_sensor':
+    return 'mdi:radiobox-marked';
```

Comment:
Since we have an optional access to a state here, we could do a similar setup here like the alarm control panel above this line.